packages/core/src/index.spec.ts

```diff
@@ -1,31 +1,32 @@
-import { debug, HoprDB } from '@hoprnet/hopr-utils'
-import HoprCoreEthereum from '@hoprnet/hopr-core-ethereum'
-import { expect } from 'chai'
+import { connectorMock } from '@hoprnet/hopr-core-ethereum'
+import { dbMock, debug } from '@hoprnet/hopr-utils'
+import chai, { expect } from 'chai'
+import chaiAsPromised from 'chai-as-promised'
 import PeerId from 'peer-id'
 import sinon from 'sinon'
-import Hopr, { HoprOptions } from '.'
+import Hopr, { sampleOptions } from '.'
 
+chai.use(chaiAsPromised)
 const log = debug('hopr-core:test:index')
 
 describe('hopr core (instance)', async function () {
-  let peerId: PeerId, db: HoprDB, chain: HoprCoreEthereum, options: HoprOptions
+  let peerId: PeerId
   beforeEach(async function () {
     peerId = await PeerId.create({ keyType: 'secp256k1', bits: 256 })
-    options = { environment: { id: '1' } } as unknown as HoprOptions
-    db = sinon.createStubInstance(HoprDB)
-    chain = sinon.createStubInstance(HoprCoreEthereum)
   })
 
   afterEach(function () {
     sinon.restore()
   })
 
-  it('should be able to create a hopr node instance without crashing', async function () {
-    expect(() => {
-      log('Creating hopr node...')
-      const node = new Hopr(peerId, db, chain, options)
-      log('Node created with Id', node.getId().toB58String())
-      expect(node instanceof Hopr)
-    }).to.not.throw()
+  it('should be able to start a hopr node instance without crashing', async function () {
+    this.timeout(5000)
```

Comment:
Why do we need to wait at the start?

Reply:
This is not a wait, but a timeout. The usual timeout for mocha tests is `3000`, I'm increasing it to `5000`